projects/angular/src/forms/styles/_radio.clarity.scss

```diff
@@ -85,7 +85,6 @@
   }
 
   .clr-form-compact .clr-radio-wrapper {
-    @include forms-mixins.form-flatten-compact-adjustments();
```

Comment:
Creating inconsistency between other form controls and radio / checkbox because of specificity being higher here and overriding what is in _containers.clarity.scss